manifests/base/deployment.yaml

```diff
@@ -1,27 +1,43 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cachemachine
+  name: moneypenny
   labels:
-    app: cachemachine
+    app: moneypenny
 spec:
   replicas: 1
   selector:
     matchLabels:
-      name: cachemachine
+      name: moneypenny
   template:
     metadata:
       labels:
-        name: cachemachine
+        name: moneypenny
     spec:
```

Comment:
If this is the live manifest that we'll be using in production, it should have the pod and container hardening settings from SQR-048 (except for disabling mounting the service account token, of course).

Reply:
done